### PR TITLE
Add lesson: introduction-to-populating-a-website-with-api-data

### PR DESCRIPTION
**Pull request by rivaquiroga**
As a new editor, I'm shadowing @acrymble on this lesson (https://github.com/programminghistorian/ph-submissions/issues/196).
Hope I've done everything as expected :crossed_fingers:
Please let me know if any change is needed.

### CODE REVIEW

**Comment by acrymble** [comment]
Looks like we have some travis errors I haven't seen before. @mdlincoln can you advise on what we've done wrong?

File: en/lessons/introduction-to-populating-a-website-with-api-data.md [hunk 1]
```diff
@@ -0,0 +1,529 @@
...
+reviewers:
+- Amber Stubbs
+- Brianna Allen
+- Adam Crymble
+editors:
+- Jessica Parr
+review-ticket: https://github.com/programminghistorian/ph-submissions/issues/196
+difficulty: 2
+activity: acquiring
+topics: apis
```
Comment (mdlincoln) [inline en/lessons/introduction-to-populating-a-website-with-api-data.md]: `topics` is like `authors` - it needs to be supplied as a list variable. In this case, please change it to

```yaml
topics:
- api
```

File: en/lessons/introduction-to-populating-a-website-with-api-data.md [hunk 2]
```diff
@@ -0,0 +1,529 @@
+---
+title: |
+    Introduction to Populating a Website with API Data
+collection: lessons
+layout: lesson
+slug: website-api
+date: 2019-05-22
+authors:
+- Go Sugimoto
+reviewers:
```
Comment (mdlincoln) [inline en/lessons/introduction-to-populating-a-website-with-api-data.md:10]: remove this duplicate `reviewers:` key

**Review by mdlincoln — DISMISSED** [review]
That warning is because `topics` wasn't properly formatted. I've added a clearer warning to our build script.

**Comment by rivaquiroga** [comment]
All checks have passed :rocket:

**Comment by acrymble** [comment]
Thanks @rivaquiroga good work. The next thing is to check if there are any problems with the tutorial and the way it is rendering. I notice from looking through that one of the links has not properly rendered. You can see it is showing the full markdown rather than the HTML link. This is the type of thing we need to catch at this stage.

Screenshot 2019-05-31 at 19 10 00

I also worry a bit that there are too many screenshots and that this will make the lesson more difficult to translate. I think it would be good practice for you to check through the lesson and make sure everything looks ok, and to suggest if there are any screenshots that could be removed without making the lesson more difficult to follow.

This is our last chance to make sure everything is ok before we publish, so please take a close look and let me know if you find anything you think we need to address.

**Comment by rivaquiroga** [comment]
@acrymble, I fixed the problem with the link not properly rendered.
In regards to screenshots, I actually found them very useful. I had never used Xampp before, so I found really helpful to have screenshots to check if everything was going as expected. It's true that some of them will need to be translated, but I'm not sure if that will really make the process more difficult, as is just taking screenshots (is not like when we have to translate a dataset or find a new one to recreate the lesson).

**Review by acrymble — DISMISSED** [review]
This looks good. I think it's ready to go once the link errors from another lesson are resolved for Travis to pass.

**Comment by acrymble** [comment]
The Travis errors blocking this merge (in other lessons) may not all actually be errors. One is:

./_site/fr/lecons/introduction-a-la-stylometrie-avec-python.html
  *  External link https://fr.wikipedia.org/wiki/Test_du_%CF%872 failed: 404 No error

But the actual link is to the page for Chi squared, which is being interpreted (even when I paste it here) as something different than it appears in the URL itself, which uses the actual Chi character.

---

- ./_site/es/lecciones/normalizar-datos.html
  *  External link http://www.diveintopython.net/regular_expressions/index.html failed: 404 No error
  *  External link http://www.diveintopython.net/xml_processing/unicode.html failed: 404 No error

Apparently this site is down for scheduled maintenance, so this should resolve itself naturally in time.

--

- ./_site/en/lessons/creating-mobile-augmented-reality-experiences-in-unity.html
  *  External link https://unity3d.com/learn/tutorials/modules/beginner/editor failed: 404 No error

I can't even find this link in the live tutorial.

**Comment by mdlincoln** [comment]
I've merged in changes from other branches that have already fixed two of the 404s. Please replace the www.diveintopython.net links with web archive links and then this can get published.

**Comment by acrymble** [comment]
I'm unable to make this change because my internet provider blocks the internet archive. Can someone else make the adjustment?

**Comment by rivaquiroga** [comment]
I'll try to do it during the afternoon. If I have any questions, I'll reach you back

**Review by mdlincoln — APPROVED** [review]

**Comment by mdlincoln** [comment]
Thank you for fixing the links @rivaquiroga - unfortunately the link checker we use does not display extra errors when a link shows up in multiple places (https://github.com/gjtorikian/html-proofer/issues/411) so when you continue to see errors even after you think a link was fixed, you should do a search over all the files in the repository to see where else the broken link appears. I fixed those other instances, and this lesson should now be ready to merge whenever you are ready.

**Comment by acrymble** [comment]
@rivaquiroga @mdlincoln thanks for helping get this published. I'm going to merge the request.

**Pull request merged**